xbmc/video/VideoInfoScanner.cpp

```diff
@@ -2285,9 +2288,14 @@ CVideoInfoScanner::~CVideoInfoScanner()
       {
         const int64_t size{pItem->GetSize()};
         digest.Update(&size, sizeof(size));
-        KODI::TIME::FileTime time{};
-        pItem->GetDateTime().GetAsTimeStamp(time);
-        digest.Update(&time, sizeof(time));
+        // linux and windows platform don't follow the same output format (linux return a zero value for milliseconds member)
```

Comment:
Can you wrap comments at 80 chars for readers with small screens or large fonts?